xml/art_sle_ha_pmremote.xml

```diff
@@ -466,12 +466,22 @@ Full list of resources:
     <title>Restrictions Regarding Groups and Constraints</title>
     <para>
      Never involve a remote node connection resource in a resource group,
-     colocation constraint, or order constraint.
+     colocation constraint, or order constraint. This may lead to unstable
+     behavior of the cluster.
     </para>
     <para>
-     <remark>taroth 2017-08-09: why? would be good to give an explanation
-     (people are more likely to follow a warning if they know why (or what can
-     happen if they disrespect the warning) :)</remark>
+     If necessary, location constraints can be configured to define the
+     relationship between a resource and a remote node.</para>
```

Comment:
This is for sure feasible though;-) I'm not sure if it's a bit confusing to mention this here, since such location constraints don't have much to do with the the special resource itself anymore.